snap/quota/quota.go

```diff
@@ -283,19 +284,26 @@ func (grp *Group) getTotalCPUPercentage() int {
 func (grp *Group) getQuotaAllocations(allQuotas map[string]*groupQuotaAllocations, skipGrp *Group) *groupQuotaAllocations {
```

Comment:
should this lose the skipGrp argugment, is not used anymore?

Reply:
Good catch, yes it can be removed